portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that gets data from a csv and parses it as to use it in a chart.*/
+@WebServlet("/chart-data")
+public class ChartServlet extends HttpServlet {
+  private LinkedHashMap<String, Integer> memeSearchTrend = new LinkedHashMap<>();
```

Comment:
nit: can you add a comment before this variable declaration describing what it is / what it's purpose is?

Reply:
Done!

Reply:
nit: make sure the comment explains more what the purpose of this variable is! For instance, "this map stores search trends for the search term 'meme' keyed by year.", or something like that!